doc/man3/DES_random_key.pod

```diff
@@ -243,6 +243,8 @@ third parameter.  This mostly emulates the normal non-thread-safe semantics
 of crypt(3).
 The B<salt> must be two ASCII characters.
 
+The values returned by DES_fcrypt() and DES_crypt() are terminated by NULL.
```

Comment:
I find this a bit ambiguous. NULL customarily refers to NULL *pointer*, while we are obviously talking about *character* here.  In other words "terminated by null character" would presumably be better. And I wouldn't be surprised if it should read "terminated by *a* null character"...

Reply:
ascii(7) clearly shows the value 0 as ```NUL '\0' (null character)```

Reply:
Yeah, but NULL, i.e. with double L, still has quite too strong pointer "clang"... In other words "NUL character" would also do, "NULL" alone ... not so much....

Reply:
Yes, NULL means pointer and NUL means \0 byte.  It would be nice to fix this, or a new PR.

Reply:
Fixed in this PR